Components/Signup/Signup.tsx

```diff
@@ -0,0 +1,236 @@
+
+import styles from './Signup.module.scss'
+function Signup({ login, setLoginKey }) {
```

Comment:
Keep the same format!

```suggestion
const Signup = ({ login, setLoginKey }) => {
```